tests/bluetooth/controller/common/defaults_cmake.txt

```diff
@@ -29,12 +29,15 @@ FILE(GLOB ll_sw_sources
 
 FILE(GLOB mock_sources
         $ENV{ZEPHYR_BASE}/tests/bluetooth/controller/mock_ctrl/src/ll.c
+        $ENV{ZEPHYR_BASE}/tests/bluetooth/controller/mock_ctrl/src/ll_assert.c
 )
 
 FILE(GLOB common_sources
         $ENV{ZEPHYR_BASE}/tests/bluetooth/controller/common/src/helper_pdu.c
         $ENV{ZEPHYR_BASE}/tests/bluetooth/controller/common/src/helper_util.c
 )
 
+set(CMAKE_BUILD_TYPE Debug)
+
```

Comment:
You can run sanitycheck with -xCMAKE_BUILD_TYPE=Debug

Reply:
Ok